### PR TITLE
Sitecore 10.1 update for TDS-based examples

### PR DESCRIPTION
**Pull request by navancommits**
sitecore 10.1 tds examples tested and work fine....

### CODE REVIEW

**Comment by navancommits** [comment]
Please reject the other PR that is based on master branch.....

File: examples/helix-basic-tds/.env [hunk 1]
```diff
@@ -16,19 +16,23 @@ SITE_HOST=www.basic-company-tds.localhost
...
+TOOLS_VERSION=10.1.0-1809
 
 SITECORE_ADMIN_PASSWORD=
 SQL_SA_PASSWORD=
 TELERIK_ENCRYPTION_KEY=
 SITECORE_IDSECRET=
 SITECORE_ID_CERTIFICATE=
 SITECORE_ID_CERTIFICATE_PASSWORD=
-SOLR_CORE_PREFIX_NAME=sitecore
+SOLR_CORE_PREFIX_NAME=basictds
```
Comment (nickwesselman) [inline examples/helix-basic-tds/.env:28]: Can we keep this consistent across the examples and just remain `sitecore`? No reason for them to be unique in containers.

**Review by nickwesselman — CHANGES_REQUESTED** [review]
Thanks so much! This is a good start, can you do the following as well? If not, I can also do in a follow-up PR.

- [ ] Add the new `SITECORE_DEVELOPMENT_PATCHES` to CM and CD. [Example](https://github.com/Sitecore/Helix.Examples/commit/6f2290c77a8302a1b243bd3f59eb94d36b38920a#diff-e00389ce3737ab7acc9be22610bf7656a3024ca2b9865476fd7857a123214251R140-R141)
- [ ] Update `PlatformVersion` in `Packages.props` in order to update NuGet references
- [ ] Change all uses of `Set-DockerComposeEnvFileVariable` in `init.ps1` scripts to the new `Set-EnvFileVariable`

**Review by nickwesselman — APPROVED** [review]
LGTM thanks so much for the contribution!

**Pull request merged**